ads/invibes.md

```diff
@@ -0,0 +1,45 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Invibes
+
+## Example
+
+```html
+<amp-ad width="300" height="300"
+    type="invibes"
+    data-pid="placement1"
+    data-ad-categ="infeed">
+</amp-ad>
+```
+
+## Configuration
+
+For configuration semantics, please contact [Invibes](https://www.invibes.com/#section-contact-email)
+
+Supported parameters:
```

Comment:
nit: Looks like those two are mandatory based on the implementation. Could we please label them as required parameters.

Reply:
Thanks for the remark! My implementation was faulty. We have a default value for those parameters on our backend so they should definitely be optional.